src/test/c_api_tests.cpp

```diff
@@ -47,11 +48,13 @@ static void testDefaultSingleModelOptions(ModelsSettingsImpl* modelsSettings) {
     {                                         \
         auto* err = C_API_CALL;               \
         if (err != nullptr) {                 \
+            uint32_t code = 0;                \
             const char* msg = nullptr;        \
+            OVMS_StatusGetCode(err, &code);   \
             OVMS_StatusGetDetails(err, &msg); \
             std::string smsg(msg);            \
             OVMS_StatusDelete(err);           \
-            ASSERT_TRUE(false) << smsg;       \
+            ASSERT_EQ(0, code) << smsg;       \
```

Comment:
This modifies the test behavior. The tests will not fail when status is not null and code is 0. This is currently impossible but still the test should protect us from such scenario